content-src/components/Sections/_Sections.scss

```diff
@@ -83,6 +83,7 @@
 
   @media (min-width: $break-point-large) {
     line-height: 16px;
+    height: 16px;
```

Comment:
This just ensures the placeholder is always the same height no matter what content it has so it doesn't shift the page.
without this the page shifts by two pixels when the cta is added.